es-ds-docs/app.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <div>
+    hello world
```

Comment:
Should probably be `<slot />`

Reply:
Primary setup of the docs site beyond "hello world" will be accomplished in a [separate ticket](https://energysage.atlassian.net/browse/CED-1725) that will immediately follow this foundational setup.